Libraries/libutil/Headers/libutil/Subprocess.h

```diff
@@ -33,7 +33,9 @@ class Subprocess {
                  std::ostream *output = nullptr,
                  std::ostream *error = nullptr)
     {
-        return execute(path, { }, { }, "", input, output, error);
+        std::vector<std::string> emptyVector;
+        std::unordered_map<std::string, std::string> emptyMap;
```

Comment:
Not really.  The only advantage of this is that the line using these variables is shorter.